edb/edgeql/compiler/setgen.py

```diff
@@ -1102,6 +1099,12 @@ def scoped_set(
     return ir_set
 
 
+def lol_set(
```

Comment:
What is this for?

Reply:
Oh oops, that was something I used while doing the migration